### PR TITLE
[VC] Fix project path

### PR DESCRIPTION
**Pull request by Fei-Guo**
This change adds "kubernetes-sigs" in the project path.

### CODE REVIEW

File: incubator/virtualcluster/pkg/apis/tenancy/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-// Code generated by deepcopy-gen. DO NOT EDIT.
+// Code generated by main. DO NOT EDIT.
```
Comment (resouer) [inline incubator/virtualcluster/pkg/apis/tenancy/v1alpha1/zz_generated.deepcopy.go]: Not sure why this is changed?

Comment (Fei-Guo) [reply incubator/virtualcluster/pkg/apis/tenancy/v1alpha1/zz_generated.deepcopy.go]: The original file was generated from controller-gen in vendor directory. We use go mod now so the controller-gen@v.0.1.1 is used which makes the difference.

**Pull request merged**